python/helpers/requirements.txt

```diff
@@ -2,7 +2,7 @@ pip>=21.3.1,<22.2.3  # Allow earlier versions to retain python 3.6 support
 pip-tools>=6.4.0,<6.8.1  # Allow earlier versions to retain python 3.6 support
 flake8==5.0.4
 hashin==0.17.0
-pipenv==2022.4.8
+pipenv>=2022.4.8,<=2022.8.5  # Allow earlier versions to retain python 3.6 support
```

Comment:
We'll probably want to tell dependabot to ignore this dependency, because I think it'll override the range here?

Reply:
It didn't on the 2 previous ones (pip and pip-tools) that I added in April.  I think it handles this correctly.